### DISCUSSION TITLE
Changing 'Axes' to be consistent spelling

### DISCUSSION THREAD

**Discussion by wrenbirda** [TheOdinProject/curriculum#24965] [General]
I'm currently working through the 'Axes' portion of the Foundation Course. I noticed that in the scrimba video, it's spelled axis (which is how I would spell axis, if in reference to an orientation) but the page is titled Axes and spelled as such throughout. I'm not far enough along to feel comfortable editing and suggesting changes to TOP yet, but I wanted to see if anyone else had noticed and if this was worth the edit. :)

**Comment by KevinMulhern** [comment]
Thanks for the suggestion @wrenbirda, however the spelling in the lesson is correct. We're using the plural "axes" to refer to both the Flexbox main axis and cross axis.

**Reply by wrenbirda** [reply]
Ah, good to know. Thank you for replying.
